File: 0002.c
/* AOJ 0002 Digit Number */

#include <stdio.h>

int
main(void)
{
    int a, b, sum;
    int n_digits;

    while (scanf("%d %d", &a, &b) == 2) {
        sum = a + b;
        n_digits = 0;

        while (sum > 0) {
            n_digits++;
            sum /= 10;
        }

        printf("%d\n", n_digits);
    }

    return 0;
}

File: 0002.rb
# AOJ 0002 Digit Number

sums = ARGF.each_line.map {|line| line.split(" ").map {|x| x.to_i} \
                           .reduce(0, :+)}
sums.map {|s|
  if s.zero?
    1
  else
    Math.log10(s).to_i + 1
  end
}.each {|d| puts d}
